src/org/zaproxy/zap/extension/scripts/CommandPanel.java

```diff
@@ -140,5 +164,32 @@ public void setAutoCompleteEnabled(boolean enable) {
             this.autocompleteListener.setEnabled(enable);
         }
     }
-	
+
+	private void setCommandScriptContent(String str) {
+		if (str != null && str.length() > EDITOR_SCRIPT_MAX_SIZE_THRESHOLD) {
+			getTxtOutput().setText("");
+			largeScriptContent = str;
+			isLargeScriptContent = true;
+		} else {
+			if (str == null) {
+				getTxtOutput().setText("");
+			} else {
+				getTxtOutput().append(str);

```

Comment:
`setText` in both cases (`null` defaults to empty and  the `append` is not required, is being called as set).